.github/workflows/pmc-ai-pr-summarizer.yml

```diff
@@ -0,0 +1,36 @@
+# PMC AI PR Summarizer Workflow
+# This GitHub Actions workflow is triggered on pull requests when they are opened or synchronized.
+# It runs on an Ubuntu-latest environment.
+# The workflow has permissions to perform write operations on PR.
+#
+# Steps:
+# 1. Checks out the repository using the actions/checkout@v4.
+# 2. Sets up Node.js version 20 using actions/setup-node@v4.
+# 3. Installs dependencies using npm ci.
+# 4. Executes the build script from the build/index.js file.
+# 5. Displays the Node.js version using node -v.
+name: PMC AI PR Summarizer
+on:
+  pull_request:
+    types: [ opened, synchronize ]
```

Comment:
Since this is a reusable workflow, this needs to follow the triggers used in the other workflows:

* https://github.com/penske-media-corp/github-workflows-wordpress/blob/52a2bfc7bb151b8200e083dcc0493197e2e29f10/.github/workflows/unit-tests.yml#L3-L4
* https://github.com/penske-media-corp/github-workflows-wordpress/blob/52a2bfc7bb151b8200e083dcc0493197e2e29f10/.github/workflows/unit-tests.yml#L11

Reply:
Done